validator/testdata/feature_tests/ads.html

```diff
@@ -664,7 +664,7 @@ <h2>Doubleclick no ad</h2>
       data-slot="/4119129/doesnt-exist">
   </amp-ad>
 
-  <h2>Doubleclick with overriden size</h2>
+  <h2>Doubleclick with overridden size</h2>
```

Comment:
I think it is easier to revert all files in validator/testdata/

Reply:
Trying with only the above reverts.
Let's see CI!
Revert the whole validator/testdata/ on failure.

Reply:
Done 🍏